k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+usage()
+{
+    echo "Usage: bash script.sh <controller_pod_name> <application_namespace> <numbber_of_snapshots_to_delete>"
```

Comment:
minor enhancement  can be done to the usage. it is usually easy for users to provide the `pv-name` than the controller pod name and its namespace etc., Similar to jiva_upgrade script, can this input be changed to take the pv-name.